src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.h

```diff
@@ -54,14 +54,18 @@ class MEDCORE_EXPORT medRegistrationSelectorToolBox : public medToolBox
 
 public slots:
 
+    /**
+     * @brief instantiates the right process toolbox according to its description
```

Comment:
The comment does not correspond to the function signature below (int takes an index, not a description). In general though, comments for virtual functions should not be copied to their overrides, because this is just creating massive code duplication (comments are code too).